source/2020-03-13-the-ember-times-issue-139.md

```diff
@@ -14,13 +14,18 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Feb 29th, 2020 - EmberJS Chennai Meetup 🇮🇳](https://www.youtube.com/playlist?list=PLh_rF0Qob_sxjhg9qPTlXn-6EZzmmTHoc)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+On February 29th 2020, EmberJS Chennai featured seven speakers at their third Meetup. They covered various topics that included:
```

Comment:
I think the Markdown lint that we have requires a space in-between. This should help with passing CI.
```suggestion
On February 29th 2020, EmberJS Chennai featured seven speakers at their third Meetup. They covered various topics that included:

```